JaimeGG64/artpreneurship-air
# AIR-018 --  Build out the about page to be responsive

# Description
This branch has the about page built out.

# Testing
1. Go to the about page
2. Use your browser responsive tool. Ensure that on different devices it looks as intended.
